Tests/ApolloCacheDependentTests/StarWarsServerTests.swift

```diff
@@ -299,7 +299,7 @@ class StarWarsServerTests: XCTestCase {
 
       let expectation = self.expectation(description: "Performing mutation")
 
-      client.perform(mutation: mutation) { (result, error) in
+      client.perform(mutation: mutation, fetchHTTPMethod: .POST) { (result, error) in
```

Comment:
I think all these ones which aren't in `FetchQueryTests` should probably just use the default rather than handing in the `.POST` parameter - glad to hear the earlier request to change this caught some issues!

Reply:
Good point! Changing it!